src/sql/workbench/contrib/charts/browser/graphInsight.ts

```diff
@@ -208,11 +208,11 @@ export class Graph implements IInsight {
 					}
 				}];
 
-				if (options.xAxisMax) {
+				if (options.xAxisMax !== undefined) {
```

Comment:
interesting 0 is considered false in JS.  I didn't realize this gem from C was in this language as well 😄.